README.md

```diff
@@ -37,28 +42,27 @@ This will be the file that holds your S3 and SQS configurations.
       }
     }
     ```
-1. [Register or create a new GitHub organization](https://github.com/settings/organizations). Find your organization's ID by visiting `https://api.github.com/orgs/<your-org-name>` and copying the `id` into the whitelist of `organizations` in `config/local.js`.
+1. [Register or create a new GitHub organization](https://github.com/settings/organizations) with a name of your choosing. Then find your organization's ID by visiting `https://api.github.com/orgs/<your-org-name>` and copying the `id` into the whitelist of `organizations` in `config/local.js`.
     ```js
     organizations: [
       99999999 // your org added here
     ]
     ```
     The organization will need to grant access to federalist, which can be done during:
       * a first-time login with your GitHub credentials
-      * in the Settings -> Applications -> federalist view in your GitHub Account
-1. Type `cf login --sso -a https://api.fr.cloud.gov -o gsa-18f-federalist -s staging` in terminal.
-1. Visit https://login.fr.cloud.gov/passcode to get a one time passcode.
-1. Enter your passcode back into terminal.
-1. Type `cf apps`.
+      * in the [Authorized OAuth Apps](https://github.com/settings/applications) tab in your GitHub Account settings
```

Comment:
this starts with a preposition (`in`) but there is already one in the initial clause (`during`)